### PR TITLE
Remove deprecated preference variable and eliminate map directory cac…

### LINKED ISSUE #1526: Error on start up
User can not start game he gets an error
Log: 
https://github.com/vicblack11/FAF/commit/490f7f6515ce44c260bd27dd75cc8fe457816ebe

### PR DESCRIPTION
**Pull request by 1-alex98**
…hing

Fixes #1526

### CODE REVIEW

**Comment by 1-alex98** [comment]
@betrok u cached  the prefence value hence new users could not create games any longer, because the value was null for them at the start of teh application :D but dont worry I will quickly hotfix it

File: src/main/java/com/faforever/client/map/MapService.java [hunk 1]
```diff
@@ -184,15 +182,14 @@ private void tryLoadMaps() {
       // TODO notify user
     }
 
-    customMapsDirectory = mapsDirectory;
```
Comment (1-alex98) [inline src/main/java/com/faforever/client/map/MapService.java:187]: there is the bad caching

**Comment by betrok** [comment]
Hmm... I don't think caching was the root of the problem. In fact it was around there before my changes.
Maps must be reloaded when the path changes, and this variable would be updated when this happens.

May be I will investigate later but idk when :/

**Comment by 1-alex98** [comment]
U are right actually

**Comment by 1-alex98** [comment]
No idea now it works before it did not

**Pull request merged**